src/SimpleCalculator/UserInputs.cs

```diff
@@ -1,17 +1,23 @@
 namespace SimpleCalculator;
 
 public class UserInputs
```

Comment:
if you are using specific use input like Console, please specify it in the class name